puppet-client.html

```diff
@@ -162,16 +199,46 @@
                 value: '/'
             },
 
+            /**
+             * @private
```

Comment:
https://www.polymer-project.org/1.0/docs/devguide/properties
I think, `readOnly: true, notify: false` is worth checking.